frontend/packages/noobaa-storage-plugin/integration-tests/tests/objectSerivceDashboard.scenario.ts

```diff
@@ -0,0 +1,106 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { untilNoLoadersPresent } from '@console/internal-integration-tests/views/crud.view';
+import { waitFor } from '@console/ceph-storage-plugin/integration-tests/utils/helpers';
+import {
+  efficiencyValue,
+  healthOfMCG,
+  noobaaCount,
+  obcCount,
+  resourceProviders,
+  savingsValue,
+  resiliencyOfMCG,
+} from '../views/noobaaDashboardPage.view';
+import { testBucket } from '../mocks/obcData';
+
+const loadAndWait = async () => {
+  await browser.get(`${appHost}/dashboards/object-service`);
+  await browser.wait(until.and(untilNoLoadersPresent));
+};
+
+describe('Tests Buckets Card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Test at least one noobaa bucket is present', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(noobaaCount));
+    const text = await noobaaCount.getText();
+    const [count] = text.split(' ');
+    expect(Number(count)).toBeGreaterThanOrEqual(1);
+  });
+
+  it('Create an Object Bucket Claim and test equality', async () => {
+    await browser.wait(until.visibilityOf(obcCount));
+    let text = await obcCount.getText();
+    const [initialCount] = text.split(' ');
+    execSync(`echo '${JSON.stringify(testBucket)}' | kubectl create -f -`);
+    await waitFor(obcCount, (Number(initialCount) + 1).toString());
+    text = await obcCount.getText();
+    const [finalCount] = text.split(' ');
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+    execSync(`echo '${JSON.stringify(testBucket)}' | kubectl delete -f -`);
+  });
+});
+
+describe('Test Status Card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Check if Multi Cloud Gateway is in a healthy state', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(healthOfMCG));
+    const color = healthOfMCG.getAttribute('fill');
+    expect(color).toBe('#486b00');
```

Comment:
Wouldn't it be nicer to import the code for the green color from https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/integration-tests/utils/consts.ts ?

Reply:
Makes sense. Thanks.